manifests/consumer/service.pp

```diff
@@ -0,0 +1,9 @@
+# Pulp Consumer Service Packages
+class pulp::consumer::service {
+  service { 'goferd':
```

Comment:
There is now a duplicate definition for the goferd service if both ```pulp::child::service``` and ```pulp::consumer::service``` are included.